package.json

```
@@ -13,17 +13,20 @@
   "scripts": {
     "export:docs": "node -r esbuild-register export/docs.ts",
     "export:overrides": "node -r esbuild-register export/overrides.ts",
+    "prettier:md": "prettier --check --parser markdown '**/*.md' --ignore-path ./LICENSE.md",
+    "prettier": "prettier --write --parser markdown '**/*.md' --ignore-path ./LICENSE.md",
```

```suggestion
    "prettier:check": "prettier --check --parser markdown '**/*.md' --ignore-path ./LICENSE.md",
    "prettier": "prettier --write --parser markdown '**/*.md' --ignore-path ./LICENSE.md",
```
Why do we ignore the LICENSE file? And why shouldn't we run prettier on all files?

---

I was trying to minimize initial changes, I can run everything. 

I was ignoring the License file because it didn't seem necessary.